tests/fixtures.py

```diff
@@ -247,3 +259,17 @@ def test_diffexp(root_path, branch_spec_diffexp, test_cluster):
     branch = cf.load(root_path, branch_spec_diffexp)
     branch.process.diffexp()
     return branch
+
+
+@pytest.fixture
+def load_test_config():
+    cf.update_config(Path(__file__).parent / "config" / "test_config.yaml")
+    return
+
+
+@pytest.fixture
+def remove_plots(root_path):  # removes all plot folders in the test tree at root_1
```

Comment:
Nice cleanup. We probably should have been doing this for the other tests, but my laptop would catch fire, so I just delete all the dirs for one final test before committing -- kind of a janky solution